### DISCUSSION TITLE
npm: way to run with --ignore-legacy-peer-deps always?

### DISCUSSION THREAD

**Discussion by broksonic21** [renovatebot/renovate#9204] [Request Help] [answered]
Github Hosted

**Describe the bug**

With package-lock v2 (npm@7) there are some cases where you always need to install with --ignore-legacy-peer-deps until packages catch up, or in some held-back version scenarios.

Is it possible to get renovate to always run in that mode to avoid artifact issues via config?

See https://github.com/broksonic21/renovate_npm_repro_without_constraints/pull/2 for an example

**Have you created a minimal reproduction repository?**

- [X] I have provided a minimal reproduction repository

**Additional context**

I know the actual resolution is to not hold back versions and/or get the dependencies to update, but in our case, we are getting the same artifact issue daily while we work through this and I'd like to avoid it.

**Comment by rarkins** [comment, marked as answer]
In future please use Discussions for questions.

The answer is that you can configure it using `.npmrc` which you commit to the repo: https://docs.npmjs.com/cli/v7/using-npm/config#legacy-peer-deps

**Reply by broksonic21** [reply]
Perfect - thanks (and confirmed that worked)
